packages/common/src/content/_fetch.ts

```diff
@@ -22,8 +22,8 @@ import { DatasetResource } from "./types";
 const shouldFetchData = (item: IItem) => {
   const type = normalizeItemType(item);
   const family = getFamily(type);
-  const dataFamilies = ["template", "solution"];
-  const dataTypes = ["Web Map", "Web Scene"];
+  const dataFamilies = ["template", "solution", "map"];
+  const dataTypes = ["Web Map", "Web Scene", "Feature Layer"];
```

Comment:
```suggestion
  const dataTypes = [
    // needed for web map/scene definition
    "Web Map", "Web Scene", 
    // needed for popup template definition
    "Feature Service"
  ];
```

Reply:
@tomwayson - I agree to drop the `dataFamilies` changes.

However, I'm confused about the difference between `Feature Service` and `Feature Layer` in this context for harvesting custom popup configurations.  

I assumed since popups are configured for the specific layers `/FeatureServer/0`, and not on the parent service `/FeatureServer`, the data type we would want to target for data fetching would be `Feature Layer`.

This is also the item type for dataset pages:

<img width="187" alt="Screen Shot 2023-05-08 at 12 34 27 PM" src="https://user-images.githubusercontent.com/3883995/236916829-1c814ddb-5cd8-4693-b500-384a842b4be8.png">